app.rb

```diff
@@ -0,0 +1,33 @@
+@morse_code = {
+  '.-' => 'A', '-...' => 'B', '-.-.' => 'C',
+  '-..' => 'D', '.' => 'E', '..-.' => 'F',
+  '--.' => 'G', '....' => 'H', '..' => 'I',
+  '.---' => 'J', '-.-' => 'K', '.-..' => 'L',
+  '--' => 'M', '---' => 'O', '.--.' => 'P',
+  '--.-' => 'Q', '.-.' => 'R', '...' => 'S',
+  '-' => 'T', '..-' => 'U', '...-' => 'V',
+  '.--' => 'W', '-..-' => 'X', '-.--' => 'Y',
+  '--..' => 'Z'
+}
```

Comment:
- [ ] the code `decode('-- -.--   -. .- -- .')` doesn't work because you didn't include the letter 'N' in your alphabet.